### ISSUE TITLE
No support for RSA PUBLIC KEY

### ISSUE THREAD

**Issue by zii** [hynek/pem#36]
e.g.
```
-----BEGIN RSA PUBLIC KEY-----
MIIBCgKCAQEAq4a0j5Za0zUrkJjWWCEXBH44L3+wWQf2VRwNA3ICUHfjwbb0aGwp
4PaqY9CS88Tzmrwn9yvfkuaxa3dTIaCaoW62C6CwdCu7O2QMFZi4H2oO9NBM2ni1
rzaKulkvZV4iPGyQSiqJqMy//DAEPVx/kyOkH9oNA62srv95gs8j83inTShhUfF6
wVtZzTKDkCiNtq9ZEXl4bJnEo4CmPhpI6AKCNxztocyKDU2rV6igIfo3UjV3U8nU
DPzjzHrBPoXbzXEgY6RkmgJwzCTIkv2BYkMwafY9ogDo4e1fY6JiwWzZlSEgScZb
VfLmAh4rZNe5PiQtDZiwZkvfSK7+Sxaa4QIDAQAB
-----END RSA PUBLIC KEY-----
```

**Comment by hynek** [comment]
fixed by 965144c82703a2a6e159e4a0b4d3d988b52dfa21

**Comment by zii** [comment]
👍
